src/components/cylc/workflow/Toolbar.vue

```diff
@@ -37,7 +37,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     </v-btn>
     <!-- title -->
     <v-toolbar-title
-      class="tertiary--text font-weight-light"
```

Comment:
This doesn't seem to do anything?

Reply:
I think it may have changed in Vuetify. It used to set the font size if I remember correctly.

Reply:
Ah! It was for color. 

![image](https://user-images.githubusercontent.com/304786/113777144-cb744400-977e-11eb-935d-18cc2671b7dc.png)

I've removed theming from Vuetify as we have another issue for that, but at some point I believe we had something similar to

```js
vuetify: {
  theme: {
    dark: true,
    themes: {
      dark: {
        primary: '#4caf50',
        secondary: '#ff8c00',
        tertiary: '#4682bf',
        accent: '#9c27b0'
      }
    }
  }
},
```

All good to remove it as we won't use the themes from Vuetify I think